File: client/src/main/java/pro/panopticon/client/sensor/impl/WarnModeSuccessrateSensor.java
package pro.panopticon.client.sensor.impl;

import pro.panopticon.client.util.NowSupplier;

/**
 * This sensor is only capable of giving warnings, and will never result in an alert to pagerduty
 */
public class WarnModeSuccessrateSensor extends SuccessrateSensor {
    public WarnModeSuccessrateSensor(int numberToKeep, Double warnLimit) {
        super(numberToKeep, warnLimit, 1.1);
    }

    WarnModeSuccessrateSensor(int numberToKeep, Double warnLimit, NowSupplier nowSupplier) {
        super(numberToKeep, warnLimit, 1.1, nowSupplier);
    }
}

File: client/src/main/java/pro/panopticon/client/sensor/impl/WarnModeSuccessrateSensor.kt
package pro.panopticon.client.sensor.impl

/**
 * This sensor is only capable of giving warnings, and will never result in an alert to pagerduty
 */
open class WarnModeSuccessrateSensor(numberToKeep: Int, warnLimit: Double) :
    SuccessrateSensor(numberToKeep, warnLimit, 1.1)
